src/ui/WebView.cpp

```diff
@@ -256,6 +267,38 @@ namespace wfl::ui
         sendRequest("whatsapp://send?phone=" + phoneNumber);
     }
 
+    bool WebView::cssFileExists(const std::string& filePath) {
+        std::ifstream file(filePath);
+        return file.good();
+    }
+
+    std::string WebView::loadCssContent(const std::string& cssFilePath) {
+        std::ifstream cssFile(cssFilePath);
+        std::string cssContent((std::istreambuf_iterator<char>(cssFile)),
```

Comment:
Please use auto

Reply:
i have no idea what "auto" is, i'm not a c++ developer, this is very improvised.

Reply:
Never mind then. I'll merge it and modify your changes when I have some time.

Reply:
It's supposed to be i.e. `auto cssFile = std::ifstream{cssFilePath};`

Reply:
yeah i think my last commit broke more than it fixed, if you want i can reverse some of the changes

Reply:
ok i'll do that